modules/account-lib/src/coin/algo/utils.ts

```diff
@@ -258,6 +259,74 @@ export class Utils implements BaseUtils {
   protected mnemonicFromSeed(seed: Uint8Array) {
     return algosdk.masterDerivationKeyToMnemonic(seed);
   }
+
+  /**
+   * Build correct fee info and fee rate for Algorand transactions.
+   *
+   // eslint-disable-next-line jsdoc/require-param-type
+   * @param tx {Object} required fields or building fee info.
+   * @param feeRate {number} required fee rate for building fee info.
+   * @returns {Object} The fee information for algorand txn.
+   */
+  getFeeData(tx, feeRate: number): any {
+    const size = this.getTransactionByteSize(tx);
+    let feeInfo = this.validateFeeInfo(size, { feeRate: feeRate, fee: undefined, baseFactor: 1 });
+    if (feeInfo.fee < ALGORAND_MINIMUM_FEE) {
+      feeInfo = this.validateFeeInfo(size, { feeRate: undefined, fee: ALGORAND_MINIMUM_FEE, baseFactor: 1 });
+      feeRate = feeInfo.feeRate;
+    }
+    return { feeInfo, feeRate };
+  }
+
+  /**
+   * Check values for feeInfo building
+   *
+   * @param size {number} required size to check.
+   * @param feeData {Object} required fees to check.
+   * @returns {Object} with fee data validated.
+   */
+  protected validateFeeInfo(size: number, { feeRate, fee, baseFactor = 1000 }): any {
+    if (!Number.isSafeInteger(size) || size < 0) {
+      throw new Error('Size must be integer positive number');
+    }
+    if (_.isUndefined(feeRate) && _.isUndefined(fee)) {
+      throw new Error('Fee rate or fee are missed');
+    }
+    let fee_value, fee_rate;
+    if (!_.isUndefined(feeRate)) {
+      if (!Number.isSafeInteger(feeRate) || feeRate < 0) {
+        throw new Error('FeeRate must be integer positive number');
+      }
+      fee_rate = feeRate;
+      fee_value = Math.ceil((fee_rate * size) / baseFactor);
+    } else {
+      if (!Number.isSafeInteger(fee) || fee < 0) {
+        throw new Error('Fee must be integer positive number');
+      }
+      fee_value = fee;
+      fee_rate = Math.round((fee / size) * baseFactor);
+    }
+    const feeObj = {
+      feeRate: fee_rate,
+      fee: fee_value,
+      size: size,
+    };
+
+    return feeObj;
+  }
+
+  /**
+   * Check if passed number is positive and integer
+   *
+   * @param number {number} Number to validate.
+   * @returns {number} Same passed number.
+   */
+  protected positiveInt(number: number): any {
```

Comment:
Please remove this method since it is no longer used.

Reply:
Sorry about that... done!